app/uk/gov/hmrc/cataloguefrontend/connector/model/Dependencies.scala

```diff
@@ -28,25 +28,64 @@ object VersionState {
   case object MinorVersionOutOfDate extends VersionState
   case object MajorVersionOutOfDate extends VersionState
   case object Invalid               extends VersionState
-  case object BobbyRuleViolated     extends VersionState
-  case object BobbyRulePending      extends VersionState
+  case class BobbyRuleViolated(violation: BobbyRuleViolation) extends VersionState
+  case class BobbyRulePending (violation: BobbyRuleViolation) extends VersionState
```

Comment:
Should the `BobbyRuleViolation` be just `BobbyRule` maybe? Might be a bit clearer to read, that a violation or warning has an inner rule, instead of a warning that has an inner violation?